examples/Koenigs_lemma.ftl

```diff
@@ -6,7 +6,7 @@ Let M,N denote a set.
 
 Let f denote a function.
 
-%%% Some preliminary set theory
```

Comment:
`%` does not create a comment line in the current edition, right?
But it is a comment token in latex mode, I assume?

Reply:
Correct, `%` is not the comment token for `.ftl` files, but it is for `.tex` files.

Reply:
See my comment on `Token.hs`